tasks/__init__.py

```diff
@@ -43,109 +43,76 @@ def requirements(ctx, dev=False):
     ctx.run(cmd, echo=True)
 
 @task
-def test_module(ctx, module=None, numprocesses=1, params=['--reruns', '1']):
+def test_module_wo_exit(ctx, module=None, params=[]):
     """Helper for running tests.
     """
     import pytest
-    if not numprocesses:
-        from multiprocessing import cpu_count
-        numprocesses = cpu_count()
-    # NOTE: Subprocess to compensate for lack of thread safety in the httpretty module.
-    # https://github.com/gabrielfalcao/HTTPretty/issues/209#issue-54090252
-    args = ['-s', '-v', '--tb=short']
-    if numprocesses > 1:
-        args += ['-n {}'.format(numprocesses), '--max-slave-restart=0']
 
+    args = ['-s', '--tb=short']
     for e in [module, params]:
         if e:
             args.extend([e] if isinstance(e, str) else e)
 
+    print('>>>  pytest args: {}'.format(args))
```

Comment:
Can this be a leftover `print` statement or this is intentional? The `>>>` prefix makes me think it may be a leftover.